### PR TITLE
feat: With `maskAllText`, mask the attributes: `placeholder`, `title`, `aria-label`

### PR DESCRIPTION
**Pull request by billyvg**
If `maskAllText` is enabled, let's mask some specific attributes: `placeholder`, `title`, and `aria-label`. I've opted to just hardcode these instead of adding them as options for now since they are only tied to `maskAllText`.

Depends on #35

### CODE REVIEW

File: packages/rrweb-snapshot/typings/types.d.ts [hunk 1]
```diff
@@ -6,50 +6,50 @@ export declare enum NodeType {
     CDATA = 4,
     Comment = 5
 }
-export declare type documentNode = {
+export type documentNode = {
```
Comment (Lms24) [inline packages/rrweb-snapshot/typings/types.d.ts:9]: No action required, just curious: Why remove the `declare` here?

**Review by Lms24 — APPROVED** [review]
LGTM!

File: packages/rrweb-snapshot/typings/types.d.ts (hunk 1 above)
Comment (billyvg) [reply packages/rrweb-snapshot/typings/types.d.ts:9]: I think this was auto generated? I'll undo and see what happens

Comment (billyvg) [reply packages/rrweb-snapshot/typings/types.d.ts:9]: Yeah not sure what happened before, but `export type` is the correct statement (it also watches others in the file for `type`).

**Pull request merged**